src/recipe/Baked.Recipe.Service.Application/Ui/AppSettings.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Baked.Ui;
+
+public class AppSettings : Dictionary<string, INamedSettings>
+{
+    public void Add(INamedSettings setting) => Add(setting.Name, setting);
+}
```

Comment:
this is overly generic.

i expect this to have `List<IPlugin>`, `List<IHandler>` etc.